lib/ansible/modules/setup.py

```diff
@@ -122,6 +122,9 @@
 # Display only facts returned by facter.
 # ansible all -m ansible.builtin.setup -a 'filter=facter_*'
 
+# Gather facts about SSH host keys (public keys) on all hosts
+# ansible all -m ansible.builtin.setup -a 'filter=ssh_host_key_dsa_public,ssh_host_key_ecdsa_public,ssh_host_key_ed25519_public,ssh_host_key_rsa_public'
```

Comment:
simpler more complete filter `filter=ssh_host_key_*_public`, but I would say this is already covered by examples above

Reply:
Thanks. Closing the PR